drivers/overlay/overlay.go

```diff
@@ -812,7 +812,19 @@ func (d *Driver) getLowerDirs(id string) ([]string, error) {
 		for _, s := range strings.Split(string(lowers), ":") {
 			lower := d.dir(s)
 			lp, err := os.Readlink(lower)
-			if err != nil {
+			// if the link does not exist, we lost the symlinks during a sudden reboot.
+			// Let's go ahead and recreate those symlinks.
+			if lp == "" && os.IsNotExist(err) {
```

Comment:
Please restructure this a bit to check for `err != nil` instead of `lp == ""`.

Reply:
Fixed